crates/ide_assists/src/handlers/extract_module.rs

```diff
@@ -958,21 +935,21 @@ fn bar() -> i32 {
 mod modname {
     use crate::thirdpartycrate::nest::SomeType;
 
-    pub(crate) struct PrivateStruct {
-        pub(crate) inner: SomeType,
+    pub struct PrivateStruct {
+        pub inner: SomeType,
     }
 
-    pub(crate) struct PrivateStruct1 {
-        pub(crate) inner: i32,
+    pub struct PrivateStruct1 {
+        pub inner: i32,
     }
 
     impl PrivateStruct {
-        pub(crate) fn new() -> Self {
+        fn new() -> Self {
```

Comment:
This should keep that new visibility I think? The issue was mainly about trait items/trait impls, since associated items for traits and trait impls are not allowed to have visibility modifiers.

Reply:
yeah, I guess I presumed it wrong